src/app/Shared/Services/Api.service.tsx

```diff
@@ -230,7 +230,7 @@ export class ApiService {
     );
   }
 
-  createRecording(recordingAttributes: RecordingAttributes): Observable<boolean> {
+  createRecording(recordingAttributes: RecordingAttributes): Observable<{ ok: boolean; status: number } | undefined> {
```

Comment:
I think we can simplify this by using: https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Maybe somethin like this `type SimpleResponse = Pick<Response, "ok", "status">` and use it here because we are extracting those from Response anyways.